Description: Upstream changes introduced in version 0.9.1-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 biniou (0.9.1-1) unstable; urgency=low
 .
   * Initial release. (Closes: #605672)
 .
 The person named in the Author field signed this changelog entry.
Author: Sylvain Le Gall <gildor@debian.org>
Bug-Debian: http://bugs.debian.org/605672

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- biniou-0.9.1.orig/Makefile
+++ biniou-0.9.1/Makefile
@@ -45,6 +45,10 @@ bdump: $(SOURCES) bdump.ml
 	ocamlfind ocamlopt -o bdump -package $(PACKS) -linkpkg \
 		biniou.cmxa bdump.ml
 
+bdump.byte: $(SOURCES) bdump.ml
+	ocamlfind ocamlc -o bdump -package $(PACKS) -linkpkg \
+		biniou.cma bdump.ml
+
 test_biniou: $(SOURCES) test_biniou.ml
 	ocamlfind ocamlopt -o test_biniou -package "$(PACKS) unix" -linkpkg \
 		biniou.cmxa test_biniou.ml
